src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/perform_interaction_request.h

```diff
@@ -53,6 +53,10 @@ namespace commands {
 class PerformInteractionRequest
     : public app_mngr::commands::CommandRequestImpl {
  public:
+  /**
+   * @brief Enum, defines which PerformInteraction response came first
+   */
```

Comment:
@YarikMamykin Can you add brief info why do we need it ?

Reply:
@AByzhynar fixed in https://github.com/smartdevicelink/sdl_core/pull/3122/commits/0190836450ab1d7e130bbb8e32315eb3325a9a68